packages/react-button/src/components/CompoundButton/renderCompoundButton.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-compose/lib/next/index';
+import { CompoundButtonState } from './CompoundButton.types';
+import { compoundButtonShorthandProps } from './useCompoundButton';
+
+/**
+ * Define the render function. Given the state of a button, renders it.
+ */
+export const renderCompoundButton = (state: CompoundButtonState) => {
+  const { slots, slotProps } = getSlots(state, compoundButtonShorthandProps);
+  const { loading, iconPosition, iconOnly } = state;
+
+  return (
+    <slots.root {...slotProps.root}>
+      {loading && <slots.loader {...slotProps.loader} />}
+      {iconPosition !== 'after' && <slots.icon {...slotProps.icon} />}
+      {!iconOnly && (
+        <slots.contentContainer {...slotProps.contentContainer}>
+          <slots.children {...slotProps.children} />
```

Comment:
Take a look at the current button. I replaced the "children" slot with "content", but defaulted children to be mixed into the content props.

Reply:
This solves the whole disparity between v0 and v7 wrt "content" vs "children"

Reply:
Fixed!